src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -401,7 +401,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		const fontInfo: ISimpleSuggestWidgetFontInfo = {
 			fontFamily: font.fontFamily,
 			fontSize: font.fontSize,
-			lineHeight: Math.ceil(1.5 * font.fontSize),
+			lineHeight: c.lineHeight,
```

Comment:
Codicons have a min size of 16px, so if this is less than that, they'll get cutoff. Our default terminal value for font, 12px, and line height, 1px, mean that the icon will be cutoff by default without the use of a multiplier like 1.5.

You can also see this happens for the editor with a small font size. By default, the editor uses `18px`, which is 1.5* our default value. 
![Screenshot 2025-01-23 at 4 50 51 PM](https://github.com/user-attachments/assets/831a9716-5555-4c9a-a3d1-1ec73306b0f6)